runtime/queries/vento/highlights.scm

```diff
@@ -0,0 +1,13 @@
+(comment) @comment @spell
```

Comment:
```suggestion
(comment) @comment
```

We don't currently have spellchecking queries like Neovim. If/when we add it though the queries will be separate from the highlight queries